phalcon/Di.zep

```diff
@@ -139,12 +136,10 @@ class Di implements DiInterface
      */
     public function attempt(string! name, definition, bool shared = false) -> <ServiceInterface> | bool
     {
-        var service;
-
         if !isset this->services[name] {
-            let service = new Service(definition, shared),
-                this->services[name] = service;
-            return service;
+            let this->services[name] = new Service(definition, shared);
```

Comment:
This will make the code slightly slower.

Reply:
The difference is minimal and I'd much rather the code be neater and less complex.

Reply:
That is fine but it represents something of a policy change and so its should be talked about.